src/ngraph_stub_ops.cc

```diff
@@ -60,6 +60,9 @@ REGISTER_NGRAPH_STUB(Name("Add")
 REGISTER_NGRAPH_STUB(Name("AvgPool")
                          .Device(ngraph_bridge::DEVICE_NGRAPH)
                          .TypeConstraint<float>("T"));
+REGISTER_NGRAPH_STUB(Name("BatchMatMul")
+                         .Device(ngraph_bridge::DEVICE_NGRAPH)
```

Comment:
I believe this now goes to the confirmation pass. @aprocter can provide better info.

Reply:
This is correct. There are three things you need to do:

(1) register type constraints in the confirmation pass
(2) register a confirmation function in the confirmation pass
(3) REGISTER_NGRAPH_STUB (but this now just takes a "string" argument)

Please check out existing patterns in `master` for details.